examples/dit/configs/training/class_cond_train.yaml

```diff
@@ -11,13 +11,13 @@ model_name: "DiT-XL/2"
 dit_checkpoint: ""
 vae_checkpoint: "models/sd-vae-ft-mse.ckpt"
 sd_scale_factor: 0.18215
-enable_flash_attention: True
+enable_flash_attention: False
 use_fp16: True
 
 # data setting
 imagenet_format: True
 data_path: imagenet_samples/images/
-dataset_sink_mode: True
+dataset_sink_mode: False
```

Comment:
Is it runnable in data sink mode?

Reply:
yes. But so far the benchmark and training is based on non-data sink model. Can be turn on together with flash attention later

Reply:
great. let's update the training throughput later with FA and DS